reference/shipping_provider.yml

```diff
@@ -751,27 +706,24 @@ components:
               customer_group_name:
                 type: string
             title: Customer Group
-            description: The Group (if any) that this customer is in. This will be default
-              to zero if the customer is not in a group or is a guest.
+            description: The Group (if any) that this customer is in. This will be default to zero if the customer is not in a group or is a guest.
```

Comment:
```suggestion
            description: The Group (if any) that this customer is in. This will default to zero if the customer is not in a group or is a guest.
```